public/javascripts/script.js

```diff
@@ -1,4889 +0,0 @@
-  "yebox.com",
-  "yehey.com",
-  "yemenmail.com",
-  "yepmail.net",
-  "yert.ye.vc",
-  "yesey.net",
-  "yifan.net",
-  "ymail.com",
-  "yogotemail.com",
-  "yomail.info",
-  "yopmail.com",
-  "yopmail.pp.ua",
-  "yopolis.com",
-  "yopweb.com",
-  "youareadork.com",
-  "youmailr.com",
-  "your-house.com",
-  "your-mail.com",
-  "yourinbox.com",
-  "yourlifesucks.cu.cc",
-  "yourlover.net",
-  "yourname.freeservers.com",
-  "yournightmare.com",
-  "yours.com",
-  "yourssincerely.com",
-  "yoursubdomain.zzn.com",
-  "yourteacher.net",
-  "yourwap.com",
-  "yuuhuu.net",
-  "yyhmail.com",
-  "z1p.biz",
-  "za.com",
-  "zahadum.com",
-  "zaktouni.fr",
-  "zeepost.nl",
-  "zetmail.com",
-  "zhaowei.net",
-  "zhouemail.510520.org",
-  "ziggo.nl",
-  "zionweb.org",
-  "zip.net",
-  "zipido.com",
-  "ziplip.com",
-  "zipmail.com",
-  "zipmail.com.br",
-  "zipmax.com",
-  "zmail.ru",
-  "zoemail.com",
-  "zoemail.org",
-  "zoho.com",
-  "zohomail.com",
-  "zomg.info",
-  "zonnet.nl",
-  "zoominternet.net",
-  "zubee.com",
-  "zuvio.com",
-  "zuzzurello.com",
-  "zwallet.com",
-  "zweb.in",
-  "zxcv.com",
-  "zxcvbnm.com",
-  "zybermail.com",
-  "zydecofan.com",
-  "zzn.com",
-  "zzom.co.uk",
-  "zzz.com",
-  "zzz.pl",
-];
-
-// Below, we have frontend form validation that prevents a user from signing up with an email address with what is likely a personal domain.
-
-function professionalEmailOnly(e) {
-  const email = document.getElementById("email").value;
-  if (filteredDomains.indexOf(email.split("@")[1]) !== -1) {
-    document.getElementById("emailError").classList.remove("invisible", "h-0");
-    document.getElementById("emailError").classList.add("pt-2");
-  } else {
-    document.getElementById("emailError").classList.add("invisible", "h-0");
-    document.getElementById("passwordError").classList.remove("pt-2");
-  }
-}
-
-// Below, we have frontend form validation that prevents a user from trying to sign up with an invalid password, enhancing security. 
-
-function passwordPass(e) {
-  const password = document.getElementById("password").value;
-  const regex = /(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}/;
-  if (password.length > 4) {
-    if (!regex.test(password)) {
-      document.getElementById("passwordError").classList.remove("invisible", "h-0");
-      document.getElementById("passwordError").classList.add("pt-2");
-    } else {
-      document.getElementById("passwordError").classList.add("invisible", "h-0");
-      document.getElementById("passwordError").classList.remove("pt-2");
-    }
-  }
-}
-
-// Below, I'm handling all of the filters on the homepage that allows users to filter all of the companies that have been reviewed by their different aspects. There's 5 different event listeners for the 5 filters. Whenever one is selected, it toggles the checkedFilter booleans and runs a function that takes in those checked filters. This allows users to combine different filters and see only those that meet all of their requirements. Neat!
```

Comment:
This is a super long comment, consider splitting it into multiple lines. You shouldn't have to scroll left to right to read comments, don't be afraid to use multiple lines